src/composables/useMyKivaHome.js

```diff
@@ -0,0 +1,51 @@
+import {
+	ref,
+	computed,
+	onMounted,
+} from 'vue';
+import { getIsMyKivaEnabled, MY_KIVA_FOR_ALL_USERS_KEY } from '#src/util/myKivaUtils';
+import { readBoolSetting } from '#src/util/settingsUtils';
+import logFormatter from '#src/util/logFormatter';
+import myKivaForAllUsersQuery from '#src/graphql/query/shared/myKivaForAllUsers.graphql';
+
+export default function useMyKivaHome(apollo, $kvTrackEvent) {
+	const redirectToMyKivaHomepage = ref(false);
+	const myKivaFlagEnabled = ref(false);
+	const userData = ref(false);
+
+	const fetchUserData = async () => {
+		await apollo.query({
+			query: myKivaForAllUsersQuery,
+		}).then(({ data }) => {
+			userData.value = data?.my ?? null;
+			myKivaFlagEnabled.value = readBoolSetting(data, MY_KIVA_FOR_ALL_USERS_KEY);
+		}).catch(e => {
+			logFormatter(e, 'useMyKivaHome composable');
+		});
+	};
+
+	const homePagePath = computed(() => {
+		return redirectToMyKivaHomepage.value ? '/mykiva' : '/';
+	});
+
+	const portfolioPath = computed(() => {
+		return redirectToMyKivaHomepage.value ? '/mykiva' : '/portfolio';
+	});
+
+	onMounted(async () => {
+		await fetchUserData();
+
+		redirectToMyKivaHomepage.value = userData.value?.id && getIsMyKivaEnabled(
```

Comment:
Changes are looking good! Quick question for validation, when testing locally, are the experiment assignments being logged like expected:
- Any page where these updated links are being used
- Only showing assignment analytics when new feature flag is enabled

Reply:
With the latest commit are experiment assignments looking correct on the pages that need to log MyKiva assignment? And the assignment is correctly not being logged when not part of the experiment bucket?

Reply:
A bit lost here, `getIsMyKivaEnabled` is to get experiment flag under those conditions is what we are expecting? I mean, not rolling out to all users. But repeat, a bit lost here...

Reply:
The last parameter is passing in the override for getting all users. Does that make sense?

Reply:
Looks like is passed to get the experiment:
https://github.com/kiva/ui/blob/e029fb84ceab9762e2a4e877c3439fae23a3b1d4/src/util/myKivaUtils.js#L144

Reply:
Right, as expected I think. Are you seeing issues with the current changeset?

Reply:
Just ensure that is expected.
Thanks clarifying!